code/__HELPERS/string_num_lists.dm

```diff
@@ -0,0 +1,16 @@
+GLOBAL_LIST_EMPTY(string_num_lists)
+
+
+///based proc that yeets a list in a glob list with an index unless it already existed, if it did just return that list. perfect for keeping 1 (one) instance. thanks rohesie very cool.
+/datum/proc/string_num_list(list/values)
```

Comment:
```suggestion
/datum/proc/string_assoc_list(list/values)
```
Now that I think about it, this works not only with numbers, but with `string -> value`, for any kind of stringifiable `value`, such as numbers, strings themselves and type paths.
Technically also for `string/path -> sting/path/number`
Could you rename the proc, global and file to reflect that, and document this for clarity?

Reply:
:(

Reply:
what would you even want it renamed to, i barely know how this works.